theme/lumo/vaadin-avatar-styles.html

```diff
@@ -31,21 +31,24 @@
 
       [part="abbr"] {
         font-family: var(--lumo-font-family);
-        font-size: 3em;
+        font-size: 2.4375em;
         font-weight: 500;
       }
 
-      :host([theme~="xlarge"]) [part="abbr"],
+      :host([theme~="xlarge"]) [part="abbr"] {
+        font-size: 2.75em;
+      }
+
       :host([theme~="large"]) [part="abbr"] {
-        font-size: 2.875em;
+        font-size: 2.4375em;
```

Comment:
Looks like `2.375em` would be a bit better here:

<img width="345" alt="Screenshot 2020-08-18 at 14 10 43" src="https://user-images.githubusercontent.com/10589913/90506349-c6fe0b00-e15c-11ea-8d05-10866c87a2a1.png">

Reply:
Good point! Updated.